### PR TITLE
Delay processing of raw command results hitting the buffer limit (#1126)

### PR DESCRIPTION
**Pull request by Valtaroth**
This pull request attempts to fix issue #1126 by delaying processing of messages hitting the BUFMAX length limit until a shorter message is received to ensure lines are complete before parsing them. I've tested it using a Docker connection into a Linux container and was able to attach to and debug a process which fails without the fix because messages are too long and cannot be parsed.

### CODE REVIEW

File: src/SSHDebugPS/CommandRunner.cs [hunk 1]
```diff
@@ -302,10 +303,20 @@ protected override void ReadLoop(StreamReader reader, CancellationToken token, b
...
+                        result += new string(buffer, 0, task.Result);
+                        if (task.Result < BUFMAX)
+                        {
+                            action(result);
+                            result = string.Empty;
+                        }
                     }
                     else
                     {
+                        if (string.IsNullOrEmpty(result))
```
Comment (gregg-miskelly) [inline src/SSHDebugPS/CommandRunner.cs:315]: >string [](http://example.com/codeflow?start=28&length=6)

You are missing a `!` here.

**Review by gregg-miskelly — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: src/SSHDebugPS/CommandRunner.cs (hunk 1 above)
Comment (Valtaroth) [reply src/SSHDebugPS/CommandRunner.cs]: Of course, thanks for pointing it out!

**Review by gregg-miskelly — APPROVED** [review]
Thanks so much for taking the time to submit this PR!

**Pull request merged**